_widget/src/components/app/url-dictionary.js

```diff
@@ -7,19 +7,27 @@ const DICTIONARY = {
   'Standard SSL Certificate': /standard/,
   'Reverse Zones': /zones/,
   'share management': /memberships/,
-  Automation: /automation/,
+  'Automation': /automation/,
```

Comment:
We could use some regression specs on these, but I think this is fine for now as they are fairly vague.